### ISSUE TITLE
Several versions of xorm included in go.sum

### ISSUE THREAD

**Issue by guillep2k** [go-gitea/gitea#8019]
I don't know if this is a problem, but I've noticed several versions of xorm in go.sum:

https://github.com/go-gitea/gitea/blob/13388258a4ef1593784e8daf97506575cc284f61/go.sum#L148-L155

Everything started when I wanted to reference this commit: [Add insert select where](https://github.com/go-xorm/xorm/pull/1401) in #7898.

I did that by editing `go.mod` like:
```
github.com/go-xorm/xorm master
xorm.io/core master
```

And then running `make vendor`.

There's been some discussion at the `#maintainers` chat about this, and it's been suggested that @lafriks  updated [github.com/lafriks/xormstore](https://github.com/lafriks/xormstore) (which [he did](https://github.com/lafriks/xormstore/releases/tag/v1.2.0)).

However, I couldn't figure out the right way to update `go.mod`, as I kept getting even more versions of `go-xorm/xorm` in my `go.sum`.

So I desisted of making any changes and report this here, so people with better understanding of this can solve it (if necessary) or simply ignore it.

**Comment by sapk** [comment]
```
GO111MODULE=on go get -u -v github.com/lafriks/xormstore@v1.2.0
GO111MODULE=on go mod tidy
GO111MODULE=on go mod vendor
git add vendor
```

**Comment by sapk** [comment]
Ho
I just see that v0.7.7 is not released yet and the ref v0.7.7-0.20190822154023-17592d96b35b point to a commit and not a release.

**Comment by sapk** [comment]
The new version in xormstore is v0.7.6 so we still have two version in use in the wait for a tag on xorm.
